docs/Coding-Guidelines/PullRequest.md

```diff
@@ -0,0 +1,17 @@
+# Pull Request
+When your code is in tip-top shape it is time to make a Pull Request (PR) into main.  A PR is a
+formal request to merge your code into main.
+
+## Building
+Before the PR can be accepted the code needs to run through the build pipeline to make sure your
+changes work with the existing code.  
+
+To run the build pipeline add the comment `/azp run`.  You do not need permission to run the
+pipeline.
```

Comment:
Incorrect. Pipeline execution is restricted to project contributors due to the "I can make a PR I never complete and get free compute cycles mining bitcoin and phoning the results to my personal account" problem. It's fair to say if you can complete a PR you can run the pipeline - both are restricted to less-than-world.